kasa/smart/modules/colormodule.py

```diff
@@ -0,0 +1,93 @@
+"""Implementation of color module."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from ...bulb import HSV
+from ...feature import Feature
+from ..smartmodule import SmartModule
+
+if TYPE_CHECKING:
+    from ..smartdevice import SmartDevice
+
+
+class ColorModule(SmartModule):
+    """Implementation of color module."""
+
+    REQUIRED_COMPONENT = "color"
+
+    def __init__(self, device: SmartDevice, module: str):
+        super().__init__(device, module)
+        self._add_feature(
+            Feature(
+                device,
+                "HSV",
+                container=self,
+                attribute_getter="hsv",
+                attribute_setter="set_hsv",
```

Comment:
I didn't test if the `set_value` works correctly if you pass a tuple to it, but I suppose it'd? We don't have a proper "type" for this at the moment, but maybe it's not that crucial for now (but worth adding a comment about it, perhaps)?

Reply:
Done